apps/authentication-portal/src/main/webapp/oauth2_consent.jsp

```diff
@@ -122,6 +125,11 @@
                                         </div>
                                     </div>
                                     <%
+                                                }
+                                            } catch (IdentityOAuth2ScopeException e) {
```

Comment:
If they can be either open id scope or auth scope why we are looping it in openIDscopes inline 115?